curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

```diff
@@ -17,439 +17,606 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+Which selector targets all elements with a `lang` attribute set to `"en-US"`?
 
 #### --distractors--
 
-Placeholder distractor 1
+```css
+[lang*="en"] { color: black; }
+```
 
 ---
 
-Placeholder distractor 2
+```css
+[lang$="US"] { color: black; }
+```
 
 ---
 
-Placeholder distractor 3
+```css
+[lang~="en"] { color: black; }
+```
 
 #### --answer--
 
-Placeholder answer
+```css
+[lang="en-US"] { color: black; }
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which selector targets `<a>` elements where the href attribute contains `"https"`?
 
 #### --distractors--
 
-Placeholder distractor 1
+```css
+a[href^="https"] { text-decoration: none; }
+```
 
 ---
 
-Placeholder distractor 2
+```css
+a[href~="https"] { text-decoration: none; }
+```
 
 ---
 
-Placeholder distractor 3
+```css
+a[href$="https"] { text-decoration: none; }
+```
 
 #### --answer--
 
-Placeholder answer
+```css
+a[href*="https"] { text-decoration: none; }
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How would you target all elements with a lang attribute that starts with `"es"` but not `"es-MX"`?
 
 #### --distractors--
 
-Placeholder distractor 1
+```css
+[lang*="es"]:not([lang="es-MX"]) { color: red; }
+```
 
 ---
 
-Placeholder distractor 2
+```css
+[lang~="es"]:not([lang="es-MX"]) { color: red; }
+```
 
 ---
 
-Placeholder distractor 3
+```css
+[lang$="es"]:not([lang="es-MX"]) { color: red; }
+```
 
 #### --answer--
 
-Placeholder answer
+```css
+[lang^="es"]:not([lang="es-MX"]) { color: red; }
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+You want to style the `<img>` elements only if their `alt` attribute does not contain the word `"thumbnail"`. Which selector would achieve this?
 
 #### --distractors--
 
-Placeholder distractor 1
+```css
+img[alt$="thumbnail"] { border: 1px solid red; }
+```
 
 ---
 
-Placeholder distractor 2
+```css
+img[alt^="thumbnail"] { border: 1px solid red; }
+```
 
 ---
 
-Placeholder distractor 3
+```css
+img[alt*="thumbnail"] { border: 1px solid red; }
+```
 
 #### --answer--
 
-Placeholder answer
+```css
+img:not([alt*="thumbnail"]) { border: 1px solid red; }
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+You're implementing an accessibility feature where `<button>` elements with a `data-tooltip` attribute should show a dotted underline. However, you want to exclude buttons where `data-tooltip` is exactly `"off"`. How would you write the selector?
 
 #### --distractors--
 
-Placeholder distractor 1
+```css
+button[data-tooltip="off"] { text-decoration: none; }
+```
 
 ---
 
-Placeholder distractor 2
+```css
+button[data-tooltip*="off"] { text-decoration: underline dotted; }
+```
 
 ---
 
-Placeholder distractor 3
+```css
+button[data-tooltip$="off"] { text-decoration: underline dotted; }
+```
 
 #### --answer--
 
-Placeholder answer
+```css
+button:not([data-tooltip="off"]) { text-decoration: underline dotted; }
```

Comment:
Would this not target button elements that have no `data-tooltip` at all?

Reply:
fixed this issue.
Thank you for pointing it out!